index.js

```diff
@@ -41,7 +54,27 @@ function getAllMovieTitles() {}
  *  getHighestMetascore(movies);
  *  //> 96
  */
-function getHighestMetascore() {}
+// default to 0 if array is empty
+// 1. with a variable of starting from the first movie 
+// loop through an array of objects 
+// 2. compare the "metascore" key value
+// 3. return the highest "metascore" value
+function getHighestMetascore(movies) {
```

Comment:
I love seeing the Pseudo code but we should get into the habit of removing it before pushing.